### ISSUE TITLE
Screenshotting components hangs when the story uses both decorator and function wrap syntax

### ISSUE THREAD

**Issue by gcazaciuc** [tsuyoshiwada/storybook-chrome-screenshot#6]
By mistake I've left the following piece of code:
```js
storiesOf('shipping-screen', module)
    .addDecorator(checkA11y)
    .addDecorator(withReadme(README))
    .addDecorator(withCode(style, 'sass'))
    .addDecorator(withScreenshot())
    .add(
        'Shipping Screen',
        withScreenshot()(() =>
            Utils.RenderWithTheme(<ShippingScreen {...props} />)
        )
    )
```
Please notice that I've setup both the decorator and also had the story wrapped in `withScreenshot`. 
When running the tests it hang up at  'Capturing components screenshots ' phase. It was easily fixed by removing the `withScreenshot` function wrapping.

I would have expected an error message to be displayed pointed to the cause of the hangup.

**Comment by tsuyoshiwada** [comment]
Hi @gcazaciuc, Thank you Issues!
Wow, it was a problem not supposed at all :scream:
I will try to issue a warning in the next release.

**Comment by tsuyoshiwada** [comment]
@gcazaciuc. Released as `0.5.0-alpha.1` :+1:

If there is no problem with the behavior, I'd like to release the version which removed `alpha`.
Please check when you have time.
